src/items/sxa-content/content/Verticals/Financial/Home/Data/Five Column CTA 2.yml

```diff
@@ -0,0 +1,13 @@
+﻿---
```

Comment:
Do we need all these three items? Let's leave the one that is used by the rendering we have at home page.